src/Controller/EnMarche/SearchController.php

```diff
@@ -76,11 +75,27 @@ public function searchCommitteesAction(Request $request)
      * @Route("/projets-citoyens", name="app_search_citizen_projects")
      * @Method("GET")
      */
-    public function searchAction(Request $request): Response
+    public function searchCitizenProjectsAction(Request $request)
     {
         $this->disableInProduction();
 
-        return new Response();
+        $request->query->set(SearchParametersFilter::PARAMETER_TYPE, SearchParametersFilter::TYPE_CITIZEN_PROJECTS);
+
+        $search = $this->getSearch($request);
```

Comment:
Did not look deeply into it, but having this line https://github.com/EnMarche/en-marche.fr/pull/1842/files#diff-77e84c6db7fe37094654486a56e642fbR126 apart from `handleRequest` seems weird, and if it could be moved inside the service, we could get rid of this private function. (Not really in your scope I know).